test/tests/api/emails.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+describe('/POST emails', ()=> {
+  it('should save message to DB', (done)=> {
+    let message = {from_email: 'ssing128@gmail', to_email: 'ssing128@gmail', body: 'i love you'};
+    request
+      .post('/emails')
+      .send(message)
+      .expect(200)
+      .end((err,res)=> {
+        if(err) return done(err);
+        res.body.should.be.a('object');
+        done(err);
```

Comment:
`done()` because it's clear that this will never be an error case.

Reply:
done.